hw2/market/forms.py

```diff
@@ -0,0 +1,13 @@
+from django import forms
+from django.forms import ModelForm
+from .models import ProductList
+
+
+class ImageForm(forms.Form):
+    image = forms.ImageField()
+
+
+class ProductListForm(ModelForm):
+    class Meta:
+        model = ProductList
+        fields = ['product']
```

Comment:
Далее. Столько форм уже не нужно. Можно сделать одну от `ProductsModel`
```suggestion
class ProductForm(ModelForm):
    class Meta:
        model = ProductsModel
        exclude = ['added_date']
        labels = {
            'name': 'Название продукта',
            'description': 'Описание',
            'price': 'Цена',
            'quantity': 'Количество',
            'image': 'Изображение',
        }
```